iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/AttachLoadOnVoltageLevel.java

```diff
@@ -0,0 +1,171 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.modification.topology;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.computation.ComputationManager;
+import com.powsybl.iidm.modification.NetworkModification;
+import com.powsybl.iidm.network.*;
+import com.powsybl.iidm.network.extensions.BusbarSectionPosition;
+
+import java.util.List;
+
+import static com.powsybl.iidm.modification.topology.TopologyModificationUtils.*;
+
+/**
+ * This method adds a new load on an existing voltage level. The voltage level should be described
+ * in node/breaker.
+ * @author Coline Piloquet <coline.piloquet at rte-france.com>
+ */
+public class AttachLoadOnVoltageLevel implements NetworkModification {
```

Comment:
Attach does not really mean anything, CreateFeeder ou CreateFeederCell ou maybe CreateBay ?